packages/datatrak-web/src/__tests__/features/Reports/Reports.test.tsx

```diff
@@ -244,10 +244,10 @@ describe('Reports', () => {
     );
     renderComponent(<Reports />);
     await userEvent.click(await screen.findByPlaceholderText('Select survey...'));
-    await userEvent.click(screen.getByText('Basic clinic data - Demo Land'));
+    await userEvent.click(await screen.findByText('Basic clinic data - Demo Land'));
```

Comment:
This looks fine. I just wanted to check - for the user everything works as expected?

Reply:
Yes, I have tested navigating the system and submitted a few responses, but I am a little nervous about it. Will we be doing regression testing after the merge?